esx_service/utils/auth_data.py

```diff
@@ -624,6 +688,8 @@ def create_tables(self):
                 vm_id TEXT PRIMARY KEY NOT NULL,
                 -- id in tenants table
                 tenant_id TEXT NOT NULL,
+                -- name of the VM being added to the tenant
+                vm_name TEXT,
                 FOREIGN KEY(tenant_id) REFERENCES tenants(id)
```

Comment:
Just asking, is the order like vm-uuid, vm-name, tenant-id or vm-uuid, tenant-id, vm-name - does it make any diff? Because everywhere the inserts are done as (vm-uuid, vm-name, tenant-id) :-)